server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+//Security Imports
+const cors = require('cors');
+const helmet = require('helmet');
+const passport = require("passport");
+//Auth not set up how I would like it done but I'm going to leave it for now so that we don't get any bugs in testing. 
+//Plan is to have this better organized to my liking by v5
+const authenticate = require('./routes/auth');
+const users = require("./routes/users");
+const auth = require("./routes/auth");
+const city = require("./routes/city");
+const profile = require("./routes/profile");
+//Still not sure what keys are doing 
+const keys = require("./config/keys");
```

Comment:
We have env addresses to keys for creating tokens, cookies, google/facebook auth and main address to MongoDB